compiler/internal/typeparams/subst/export.go

```diff
@@ -0,0 +1,49 @@
+// Package subst is an excerpt from x/tools/go/ssa responsible for performing
```

Comment:
I wonder if it might make sense to store these sorts of files in a directory that makes it obvious that it's a vendored/forked upstream file/excerpt.  I've noticed that the `gofumpt` project does [something similar](https://github.com/mvdan/gofumpt/tree/master/internal/govendor), which can make it easier to know where to update files from upstream, as well as a nice indicator to humans and tools (such a linters) not to modify such files.

Reply:
Good idea. Created `internal/govendor` and moved `subst` into it.